test/TestDNSSEC.js

```diff
@@ -5,6 +5,7 @@ const types = require('dns-packet/types');
 
 var dnssec = artifacts.require("./DNSSECImpl");
 const Result = require('@ensdomains/dnsprovejs/dist/dns/result')
+const { exception } = require('@ensdomains/test-utils')
```

Comment:
`exception` is a bit  too generic, can we use something else?

Reply:
do you have a better name?